beacon_node/lighthouse_network/src/discovery/enr.rs

```diff
@@ -188,6 +189,18 @@ pub fn build_enr<E: EthSpec>(
         builder.udp6(udp6_port.get());
     }
 
+    // Add EIP 7636 client information
+    if config.send_eip_7636_info {
```

Comment:
We have `private` flag in the `Config` that determines whether to send client information to peers. This flag can be used instead of adding a new flag.

https://github.com/sigp/lighthouse/blob/8124e5ca5a1f6b67a5137bfa25be18285d7b80ee/beacon_node/lighthouse_network/src/config.rs#L118-L120